### PR TITLE
Add fix 1.4.2 features

### PR DESCRIPTION
**Pull request by akegaviar**
Fix Corda 4.3 standalone shell, explorer, combined cordapps.
Add node and network statuses.
Add Quorum 2.4.0 feature -- learner nodes.

### CODE REVIEW

File: docs/platform/view-node-and-network-status.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# View node and network status
```
Comment (jona-sc) [inline docs/platform/view-node-and-network-status.md]: This bit is missing some things. 

1. We have `Error` node status as well which means the node is down and/or unreachable. 
2. Network `Error` status means it contains a node that is either `Failed` or `Error`.
3. Node `Failed` status means the node failed to deploy, while `Error` means there was an error during the lifetime of the node.

File: docs/operations/corda/node-explorer.md [hunk 2]
```diff
@@ -13,8 +13,8 @@ With the Corda Node Explorer, you can:
 To connect to your node using the Node Explorer, you will need Corda's default Finance CorDapp installed on the node that you want to explore. This enables basic cash issuance and payment transactions.
 
 1. Download the Corda Finance CorDapp JARs from Corda artifactory:
```
Comment (jona-sc) [inline docs/operations/corda/node-explorer.md]: We should replace the Finance JARs with the [tokenSDK](https://github.com/corda/token-sdk) as it is more advanced and doesn't suffer the same issues of having to install the workflows before the contracts. 

> The token SDK is intended to replace the "finance module" from the core Corda repository.

Not sure if there is a built .jar file in Corda artifactory though..

File: docs/platform/view-node-and-network-status.md (hunk 1 above)
Comment (akegaviar) [reply docs/platform/view-node-and-network-status.md]: Fixed. Thanks!

File: docs/operations/corda/tools.md [hunk 3]
```diff
@@ -35,14 +35,14 @@ See also [View node access and credentials](/platform/view-node-access-and-crede
...
-$ java -jar corda-tools-shell-cli-4.1.jar --host=nd-123-456-789.rg-123-456.p2pify.com --port=10201 --user=admin --password=pass --cordapp-directory=⁨/home/user/cordapps
+$ java -jar corda-tools-shell-cli-4.3-all.jar --host=nd-123-456-789.rg-123-456.p2pify.com --port=10201 --user=admin --password=pass --cordapp-directory=⁨/home/user/cordapps
 
    ______               __
   / ____/     _________/ /___ _
  / /     __  / ___/ __  / __ `/
 / /___  /_/ / /  / /_/ / /_/ /
 \____/     /_/   \__,_/\__,_/
---- Corda Open Source 4.1 (c11f6c1) ---
+--- Corda Open Source 4.3 (c11f6c1) ---
```
Comment (easeev) [inline docs/operations/corda/tools.md]: >c11f6c1
Will be different for 4.3

File: docs/operations/corda/tools.md [hunk 4]
```diff
@@ -88,7 +88,7 @@ $ docker run -it -v /home/user/cordapps:/cordapps chainstack/corda-shell --host=
...
```
Comment (easeev) [inline docs/operations/corda/tools.md]: Same as above

File: docs/operations/corda/tools.md (hunk 3 above)
Comment (d2phap) [reply docs/operations/corda/tools.md]: It's `d679784`   ----> https://circleci.com/gh/chainstack/e2e-tests/2501

File: docs/blockchains/quorum.md [hunk 5]
```diff
@@ -31,12 +31,13 @@ The Raft consensus algorithm has the following key characteristics:
 
 Crash fault tolerance assumes that there are no adversary nodes on the network.
 
-A Quorum Raft network always has the nodes in one of the following roles:
+A Quorum Raft network has the nodes in one of the following roles:
 
 * Minter — in Raft terminology, this is the *leader* node. In Quorum Raft terminology, this is the *minter* node.
 * Verifier — in Raft terminology, this is the *follower* node. In Quorum Raft terminology, this is the *verifier* node.
+* Learner — this is the *learner* node in both Raft and Quorum Raft terminology.
```
Comment (easeev) [inline docs/blockchains/quorum.md]: Inconsistent capitalization, Leader, follower, Learner

File: docs/platform/view-node-and-network-status.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+You can view the node and network status in the **Status** column of your nodes list or your networks list.
+
+| Node status | Network status | Description                              |
```
Comment (easeev) [inline docs/platform/view-node-and-network-status.md]: This table applies only to consortium networks as in public networks the network status is always OK. Perhaps worth making a note on that.

Comment (akegaviar) [reply docs/platform/view-node-and-network-status.md]: Split it in two: consortium and public. Please check

**Pull request merged**